tests/cost_report/infrastructure/conftest.py

```diff
@@ -55,3 +60,18 @@ def reinforcement_profile_builder(
         )
 
     yield reinforcement_profile_builder
+
+
+@pytest.fixture(name="surroundings_infrastructure_fixture")
+def _get_surroundings_infrastructure_fixture(
+    point_surroundings_for_zones_builder_fixture: tuple[
+        Callable[[], PointSurroundings], PointSurroundingsTestCase
+    ],
+) -> Iterable[SurroundingsInfrastructure]:
+    _builder, test_case = point_surroundings_for_zones_builder_fixture
+    yield SurroundingsInfrastructure(
+        infrastructure_name="dummy infrastructure",
+        # To simplify A / B total areas, we just set it to `1`.
+        infrastructure_width=1,
```

Comment:
See previous comment

Reply:
I will add a parametrized test for the previous calculator to cover this.

Reply:
Found a bug thanks to this!